includes/Checker/Checks/No_Unfiltered_Uploads_Check.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Checks\No_Unfiltered_Uploads_Check
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker\Checks;
+
+use WordPress\Plugin_Check\Checker\Check_Result;
+
+/**
+ * Check for detecting "ALLOW_UNFILTERED_UPLOADS" constant in plugin files.
+ *
+ * @since n.e.x.t
+ */
+class No_Unfiltered_Uploads_Check extends Abstract_File_Check {
+
+	/**
+	 * Check the "ALLOW_UNFILTERED_UPLOADS" constant in file.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param Check_Result $result The Check Result to amend.
+	 * @param array        $files  Array of plugin files.
+	 */
+	protected function check_files( Check_Result $result, array $files ) {
+		$php_files = self::filter_files_by_extension( $files, 'php' );
+		$file      = self::file_str_contains( $php_files, 'ALLOW_UNFILTERED_UPLOADS' );
```

Comment:
This should be a check to if is `define( 'ALLOW_UNFILTERED_UPLOADS` exists right? You could check to see if it is defined or use the constant right?